src/_pytest/legacypath.py

```diff
@@ -401,26 +401,10 @@ def Node_fspath_set(self: Node, value: LEGACY_PATH) -> None:
 
 
 @pytest.hookimpl
-def pytest_configure(config: pytest.Config) -> None:
+def pytest_load_initial_conftests(early_config: pytest.Config) -> None:
```

Comment:
Changing to `pytest_load_initial_conftests` to monkey patch the attributes as it executes earlier than `pytest_configure`.